ipasetup.py.in

```diff
@@ -19,20 +19,21 @@ import sys
 
 
 PACKAGE_VERSION = {
-    'cryptography': 'cryptography >= 0.9',
+    'cryptography': 'cryptography >= 1.5',
     'dnspython': 'dnspython >= 1.11.1',
-    'gssapi': 'gssapi > 1.1.2',
+    'gssapi': 'gssapi >= 1.2.0',
```

Comment:
I want to be consistent between RPM and pip, therefore I require the same versions as are available in Fedora 24:
```
$ rpm -qa python-gssapi python2-cryptography
python2-cryptography-1.5.3-3.fc24.x86_64
python-gssapi-1.2.0-1.fc24.x86_64
```